core/src/main/java/nl/nn/adapterframework/parameters/Parameter.java

```diff
@@ -527,10 +528,10 @@ public Object getValue(ParameterValueList alreadyResolvedParameters, Message mes
 				if (result instanceof Message && StringUtils.isNotEmpty(getContextKey())) {
 					result = ((Message)result).getContext().get(getContextKey());
 				}
-				if (log.isDebugEnabled() && (result==null ||
-					result instanceof String  && ((String)result).isEmpty() ||
-					result instanceof Message && ((Message)result).isEmpty())) {
-						log.debug("Parameter ["+getName()+"] session variable ["+requestedSessionKey+"] is empty");
+				if ((result == null) ||
```

Comment:
misschien is hier die Log.isDebugEnabled wel handig, fail-fast anders moet hij altijd al die checks doen.